index.js

```diff
@@ -2,3 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
+
+module.exports.sign = require('./lib/sign');
```

Comment:
I don't understand why you need this. In your script you should be able to just do this:
````
var sign = require('jpm/lib/sign');
````
Did you try that?